src/main/java/org/broadinstitute/hellbender/tools/copynumber/FilterIntervals.java

```diff
@@ -304,19 +307,28 @@ private void validateArguments() {
         CopyNumberArgumentValidationUtils.validateOutputFiles(outputFilteredIntervalsFile);
     }
 
-    private void resolveAndValidateIntervals() {
+    private static Pair<SimpleIntervalCollection, AnnotatedIntervalCollection> resolveAndValidateIntervals(
```

Comment:
Does this need to be `static`? Seems like we could avoid some verbosity with the inputs/outputs here.

Reply:
I actually changed it to a static method because I thought it would be cleaner! I wanted to validate that both sets of intervals line up at least once to avoid future regressions, and doing a quick check at the end of this method right before returning them as essentially final seemed minimal (albeit a bit redundant). But leaving this method as non-static and merely assigning the sets of intervals to non-final private fields (as was originally done) allows the possibility of accidental reassignment and invalidation of that assumption, however unlikely. I do wish multiple returns were more elegant and natural in Java.